google-http-client/src/main/java/com/google/api/client/http/apache/ApacheHttpTransport.java

```diff
@@ -278,7 +277,7 @@ public HttpClient getHttpClient() {
      * </p>
      *
      * <pre>
-       setProxy(new HttpHost("127.0.0.1", 8080))
+     setProxy(new HttpHost("127.0.0.1", 8080))
```

Comment:
Within a `<pre>` block, you can still include the stars (`*`) at the beginning of the line (for a more consistent look), and they are stripped out before generating the javadoc output.